spec/meilisearch/index/base_spec.rb

```diff
@@ -2,11 +2,11 @@
 
 RSpec.describe MeiliSearch::Index do
   it 'fetch the info of the index' do
-    client.create_index!('new_index')
```

Comment:
We also have some unclear names in the `indexes_spec.rb` and `documents_spec.rb`, can you update them as well?

Reply:
@brunoocasali good call! It's done! ✅